### DISCUSSION TITLE
Impressed by 2.5.0RC2 but was unsure how to --resync

### DISCUSSION THREAD

**Discussion by RedSnt** [abraunegg/onedrive#2766] [General]
After installing [Activate Linux](https://github.com/MrGlockenspiel/activate-linux), I had the bright idea to finde a solution for OneDrive, and found this project pretty easily, but was initially disappointed in the slow download speed.  
Went through the hoops of compiling rc2 on MX Linux 23.3 (debian bookworm based) and oh wow, it's absolutely blazing!  
In fact, so fast that OneDrive_Tray couldn't keep up: 
![image](https://github.com/user-attachments/assets/e2b19f6f-cbb3-44b0-b5bc-b323bed8b66b)
And managed 2.5 GB in a minute on a 1 gigabit connection, plenty fast, especially compared to the ~2mb/s it used to run at.   

How I tested it was moving a folder out of the local dropbox folder, and when I tried to sync it assumed I had deleted it and was trying to do a "big delete" in the cloud but giving me the `classify_as_big_delete` error and I spent way too long figuring out how to 'clear the queue'. That's why I'm writing this, because it would be nice if the error message mentioned how to do --resync instead of just --force, because I wasn't interested in deleting the cloud data, just redownloading the local.  

Anyway, great improvement with 2.5.0RC2, it's a real pleasure to use.

**Comment by abraunegg** [comment]
@RedSnt 
Thanks for the positive feedback re RC2. Given that you were able to compile, can you give #2728 a build and use that? This is the base for RC3 and has many fixes and subtle improvements on RC2. Your feedback on that PR would be greatly appreciated.

> That's why I'm writing this, because it would be nice if the error message mentioned how to do --resync instead of just --force, because I wasn't interested in deleting the cloud data, just redownloading the local.

Thanks for the feedback regarding this.

**Reply by RedSnt** [reply]
> @RedSnt Thanks for the positive feedback re RC2. Given that you were able to compile, can you give #2728 a build and use that? This is the base for RC3 and has many fixes and subtle improvements on RC2. Your feedback on that PR would be greatly appreciated.

Compiled #2728 without errors and downloaded from onedrive cloud just fine with the exception of a single HTTP code 503 midway.  
I don't use OneDrive much to be fair, mostly a cloud space for my KeePassXC DB backups these days. I'll be happy to report any bugs I might find though, just unlikely I stress it enough to find many bugs.
